src/schema.graphql

```diff
@@ -11,8 +11,11 @@
 type Query {
   getGroups(where: GroupSearchInput!): [Group]
   getUsers(where: UsersSearchInput!): [User]
-  getPosts(where: PostsSearchInput!): [Post]
-  getComments(where: CommentsSearchInput!): [Comment]
+  getPosts(where: PostsSearchInput!, orderBy: PostOrderByInput): [Post]
```

Comment:
Max if you use PostOrderByInput is this imported from the database model or is it defined as it does not show up in the changes